.gitignore

```diff
@@ -8,6 +8,11 @@ pip-selfcheck.json
 sgf
 pyvenv.cfg
 .DS_store
+logs/
+
+# virtualenv folder
+minigo/
```

Comment:
Interesting, my virtualenv got installed in ./bin/ and other misc. files. And I think Andrew's gets installed in ~/.virtualenvs/minigo. I'm starting to wonder how many virtualenv/venv/pipenv variants there are out there...

Reply:
Haha I see. I followed [this link](http://docs.python-guide.org/en/latest/dev/virtualenvs/#lower-level-virtualenv) from the README which just tells you to run `virtualenv my_project`, so naturally I named it `minigo`.